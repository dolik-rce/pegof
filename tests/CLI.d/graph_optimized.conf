graph
optimize concat-strings
input CLI.d/graph.peg
