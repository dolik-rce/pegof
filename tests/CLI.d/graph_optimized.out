digraph "CLI.d/graph.peg (optimized)" {
    labelloc = "t";
    label = "CLI.d/graph.peg (optimized)";
    file -> _
    file -> object
    file -> array
    object -> pair
    object -> _
    pair -> _
    pair -> string
    pair -> value
    array -> value
    array -> _
    value -> _
    value -> object
    value -> array
    value -> boolean
    value -> number
    value -> string
    value -> null
}
